Formula/unshield.rb

```diff
@@ -21,6 +22,8 @@ class Unshield < Formula
   uses_from_macos "zlib"
 
   def install
+    # cmake check for libiconv will miss the OS library without this hint
+    ENV.append "LDFLAGS", "-liconv"
```

Comment:
```suggestion
    ENV.append "LDFLAGS", "-liconv" if OS.mac?
```
I think you need this.

Also, update the test, maybe?

Reply:
Good call. I pushed a change that does test that iconv support worked. If that fails on Linux, I'll add a Linux dep on the Homebrew libiconv.

Reply:
Looks like it's happy.